utils/getEventId.js

```diff
@@ -0,0 +1,4 @@
+function getEventId(message){
+    return traits.event_id || context.traits.event_id || properties.event_id || messageId;
```

Comment:
message is passed but `traits.event_id` is referenced, but we are not extracting traits hence traits, context, properties and messageId will always be undefined

Reply:
Committed new changes, please check once